models/reviews.js

```diff
@@ -14,3 +14,25 @@ exports.fetchReviewFromId = (id) => {
       return review;
     });
 };
+
+exports.fetchReviews = () => {
+  return db
+    .query(
+      `
+  SELECT reviews.*, count(comments.review_id)::INTEGER AS comment_count FROM reviews
+  LEFT JOIN comments ON (reviews.review_id = comments.review_id)
+  GROUP BY reviews.review_id
+  ORDER BY reviews.created_at;
+  `
+    )
+    .then(({ rows }) => {
+      const reviews = rows;
+      if (!reviews.length) {
+        return Promise.reject({
```

Comment:
not convinced that this error has been tested for, or indeed whether this is a useful point at which to throw an error; 
- Reflect upon how this might be app-breaking
- Consider deletion
- If you're keeping it, it should be tested